de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/location/LocationRepository.java

```diff
@@ -38,9 +41,14 @@ private Location toLocation(@NonNull final I_C_Location locationRecord)
 	{
 		final String address = Services.get(ILocationBL.class).mkAddress(locationRecord);
 
+		Optional<I_C_Country> countryRecord = Optional.ofNullable(load(locationRecord.getC_Country_ID(), I_C_Country.class));
```

Comment:
why not using ICountryDAO.getById in this case?

PS: not the best approach in terms of design, but at least easy to track in future